remotes/docker/schema1/converter.go

```diff
@@ -435,7 +435,7 @@ func (c *Converter) schema1ManifestHistory() ([]ocispec.History, []digest.Digest
 	}
 
 	history := make([]ocispec.History, len(m.History))
-	diffIDs := []digest.Digest{}
+	diffIDs := make([]digest.Digest, 0, len(m.History))
```

Comment:
This change isn't worth making. The `len(m.History)` will almost always be larger than `len(diffIDs)` and in some cases quite a bit larger. On the other hand, `len(diffIDs)` will usually be a pretty small array. Relying on the slice growth algorithms makes more sense here.